src/flypad-backend/src/Aircraft/AircraftProcedures.cpp

```diff
@@ -0,0 +1,19 @@
+#include "AircraftProcedures.h"
+
+
+std::span<const ProcedureStep*> AircraftProcedures::getProcedure(int64_t pID) const {
```

Comment:
not sure adding a class for the procedures is an improvement

Reply:
You can't return an `std::array`, the only way to return a `std::array` is to return a `std::span`, or a `std::pair` containing the beginning of the array and the ending of the array (or the size). Returning a `std::span` is more readable than returning a `std::pair` thus `std::span` it is. Also, it's part of the standard library so technically, I'm not adding anything.

Reply:
Bad comment from my side - I meant adding a cpp file just for this

Reply:
Bad comment from my side - I meant adding a cpp file just for this

Reply:
Ah check, yes agreed. I sort of forgot about because during development, there used to be more functions in the cpp/h files which later on have been (re)moved